libs/client-api/src/http_publish.rs

```diff
@@ -205,4 +249,24 @@ impl Client {
       .await?
       .into_data()
   }
+
+  pub async fn get_published_view_reactions(
+    &self,
+    view_id: &uuid::Uuid,
+    comment_id: &Option<uuid::Uuid>,
+  ) -> Result<Reactions, AppResponseError> {
+    let url = format!(
+      "{}/api/workspace/published-info/{}/reaction",
+      self.base_url, view_id
+    );
+    let url = if let Some(comment_id) = comment_id {
+      format!("{}?comment_id={}", url, comment_id)
```

Comment:
I'd prefer to use the `query` method before the `send` on the request http builder to pass in `comment_id` there instead of appending to url